### PR TITLE
Moving css files into correct folder

### PR DESCRIPTION
**Pull request by kermy19800**
I moved Students.css and StudentCard.css into the same folder as their js counterparts.

### CODE REVIEW

File: src/components/StudentCard.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 .info-div b {
     font-size: 30px;
+    text-transform: uppercase;
```
Comment (kermy19800) [inline src/components/StudentCard.css]: Basic refactoring

Comment (Tumne) [reply src/components/StudentCard.css]: What the fuck! Why'd you do this????

Comment (kermy19800) [reply src/components/StudentCard.css]: Redundant selector

Comment (Tumne) [reply src/components/StudentCard.css]: Oh I see, my bad, good stuff

File: src/components/StudentCard.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import '../StudentCard.css';
+import './StudentCard.css';
```
Comment (Tumne) [inline src/components/StudentCard.js]: I would prefer if we included the folder name too:

`import '../components/StudentCard.css';`

File: src/components/Students.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import StudentCard from './StudentCard';
 
 
-import '../Students.css';
+import './Students.css';
```
Comment (Tumne) [inline src/components/Students.js]: Same here, should be:

`import '../components/Students.css';`

**Comment by Tumne** [comment]
Okay, that's totally acceptable, if you do need to use it for the future, please comment out that piece of code and add a `// TODO: [my-reason]` above the code.

**Comment by kermy19800** [comment]
Made all the changes you requests, ready for review

File: src/components/StudentCard.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ class StudentCard extends React.Component {
     render(){
 
         const { student, toggleGrades, appendTag, } = this.props;
+        // TODO: use the searchTag var to set state
         const { searchTag } = this.state;
```
Comment (Tumne) [inline src/components/StudentCard.js]: Can you comment out `line 18` too:
```
// TODO: use the searchTag var to set state
// const { searchTag } = this.state;	
```

and I think it'll be ready to go

Comment (kermy19800) [reply src/components/StudentCard.js]: i did it

**Pull request merged**